compile_proto.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2020-2021 Intel Corporation
```

Comment:
In the interest of not having too many top level files, this file should either stay where it is (`/openfl/protocols`), or be moved to the `/scripts` folder with other bash scripts used for development / packaging purposes.

Also, it's not needed for this PR, but as a general comment it would be good to have documentation explaining when developers need to run this file (i.e. when submitting PR's that modify existing protocols)

Reply:
complie_proto.sh was returned to openfl/protocols directory with correct build paths. Also I added README file to that directory.